lib/constructs/provisioning-sfn-workflow.ts

```diff
@@ -0,0 +1,113 @@
+import * as sfn from "aws-cdk-lib/aws-stepfunctions";
+import * as lambdaNodeJs from "aws-cdk-lib/aws-lambda-nodejs";
+import * as logs from "aws-cdk-lib/aws-logs";
+import { Construct } from "constructs";
+import { mapTasks, TasksMap } from "./sfn-lambda-invoke-task";
+
+/**
+ * Successful condition check
+ */
+export const successResponse = sfn.Condition.numberEquals("$.cspResponse.Payload.statusCodeFn", 200);
+
+/**
+ * Client error condition check∂
```

Comment:
Strange char snuck in here at the end of the line

Reply:
Finger slip maybe, removed.